src/main/java/seedu/contax/logic/commands/FindAppointmentCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.contax.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.contax.commons.core.GuiListContentType;
+import seedu.contax.commons.core.Messages;
+import seedu.contax.model.Model;
+import seedu.contax.model.appointment.ContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all appointments in address book whose name contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindAppointmentCommand extends Command {
+    public static final String COMMAND_WORD = "findappointment";
+
+    public static final String MESSAGE_USAGE = "`" + COMMAND_WORD + "`: Finds all appointments with"
+            + "names or person names that "
+            + "contain any of the specified keywords (case-insensitive) "
+            + "and displays them as a list with index numbers.\n"
+            + "Parameters: *KEYWORDS [MORE_KEYWORDS]... [by/person]*\n"
+            + "`Example: " + COMMAND_WORD + "Contract Alice Meeting`";
```

Comment:
Minor issue, but the monospaced quote is being opened too early, should be moved to after Example.
```suggestion
            + "Example: `" + COMMAND_WORD + "Contract Alice Meeting`";
```

Reply:
Fixed